hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/schema/BaselineMetricInterval.java

```diff
@@ -0,0 +1,21 @@
+package org.hypertrace.graphql.metric.schema;
+
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+
+import java.time.Instant;
+
+@GraphQLName(BaselineMetricInterval.BASELINE_METRIC_INTERVAL)
+public interface BaselineMetricInterval extends BaselineMetricAggregationContainer {
```

Comment:
A `BaselineMetricAggregationContainer` contains both baseline and non-baseline data (see my comment on its naming) - which means both would show up on the series. This doesn't match the API we had discussed, where we settled on splitting the series (which we've _also_ done here).

Reply:
BaselinedMetricAggregationContainer doesn't contains both baseline and non baseline data. It contains 3 fields for each aggregation function, Value, lowerBound and UpperBound.

Reply:
<img width="1611" alt="Screenshot 2021-01-08 at 1 29 25 PM" src="https://user-images.githubusercontent.com/73101820/103989340-9410ee00-51b5-11eb-83a8-62dd21a69cce.png">
Attached screenshot of schema

Reply:
You are right, fixed the schema

Reply:
<img width="1347" alt="Screenshot 2021-01-08 at 2 19 09 PM" src="https://user-images.githubusercontent.com/73101820/103994059-7e52f700-51bc-11eb-9f03-899c9b764c33.png">
New schema